__tests__/BleManager.js

```diff
@@ -313,7 +316,7 @@ test('BleManager properly handles errors while monitoring characteristic values'
   Native.BleModule.emit(Native.BleModule.ReadEvent, [nativeOperationCancelledError, { id: 'a', value: 'b' }, 'x'])
   subscription.remove()
   expect(listener.mock.calls.length).toBe(1)
-  expect(listener.mock.calls[0][0].message).toBe('Operation was cancelled')
+  expect(listener.mock.calls[0][0].message).toBe(BleErrorCodeDescription[BleErrorCode.OperationCancelled])
```

Comment:
Maybe it should be changed from `BleErrorCodeDescription` to `BleErrorCodeMessage` or `BleErrorCodeErrorMessage`?